searx/static/themes/simple/src/less/style.less

```diff
@@ -394,6 +382,16 @@ article[data-vim-selected].category-social {
   padding: 10px 0 0 0;
 }
 
+.result-videos {
+  img.thumbnail {
+    .ltr-float-left();
+    padding-top: 0.6rem;
+    .ltr-padding-right(1rem);
+    width: 20rem;
+    height: unset; // remove heigth value that was needed for lazy loading
+  }
+}
```

Comment:
> various API from the video engines use the term "thumbnails".
> I'm not sure how to resolve this confusion.

The intention of this PR is to remove the confusion by removing `img.img_src` .. to show video thumbnail in the old size (as shown in your screenshot) this CSS was added for results of the `video.html` template.  

Not sure, could it be that you mixed `img.img_src` and `img.thumbnail` .. sorry if my commit message was not clear, but this PR removes `img.img_src` from the results of the `default.html` template.

Reply:
Ah sorry, I have not seen this one. It makes sense.